recipes/libsequence/build.sh

```diff
@@ -2,5 +2,7 @@
 
 CPPFLAGS="-I$PREFIX/include $CPPFLAGS" LDFLAGS="-L$PREFIX/lib $LDFLAGS" ./configure --prefix=$PREFIX
 make
+export LD_LIBRARY_PATH=-L$HOME/$PREFIX/lib
```

Comment:
I have two problems with it. I think the syntax is wrong, shouldn't this be `=-L$HOME/:$PREFIX/lib`? And if you don't increase the build number of a recipe this will not be rebuild so your recipe is not using this modified version as it was not build.